src/barkhaPJun20/ArmStrongNum.java

```diff
@@ -0,0 +1,25 @@
+package barkhaPJun20;
+
+public class ArmStrongNum {
+	
+	void findArmStrongNum(int originalNumber) {
+		int number=originalNumber;
+		int sum=0;
+		while(number>0) {
+			int lastnum=number%10;           //3
+			sum=sum+(lastnum*lastnum*lastnum);
```

Comment:
Use math function here, like Math.pow(number%10, 3);
And never initialize variable in loops. like int lastnum = ; this.